app/models/picture.rb

```diff
@@ -0,0 +1,31 @@
+class Picture < ActiveRecord::Base
+
+  belongs_to :user
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+  validates :title, :image, presence: true
+
+  has_attached_file :image,
+                    :styles => {  :medium => "800x800>",
+                                  :thumb => "200x200>" },
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)